### ISSUE TITLE
Is source code available?

### ISSUE THREAD

**Issue by marcomilon** [coderatio/phpconsole#10]
Is source code available?

Thanks

**Comment by coderatio** [comment]
Yes it is. When you've installed the app, you have access to the source code and make changes to it the way you want.
